.github/workflows/linkcheck.yml

```
@@ -0,0 +1,14 @@
+name: Link checker
```

```suggestion
name: Markdown link check
```
And name the file markdown-link-check.yaml.

---

@nwt: Ok, changes made.